test/supavisor/monitoring/tenant_test.exs

```diff
@@ -35,37 +35,40 @@ defmodule Supavisor.PromEx.Plugins.TenantTest do
     end
   end
 
-  describe "execute_client_connections_lifetime" do
-    setup ctx do
-      create_instance([__MODULE__, ctx.line])
-    end
+  # for {key, port} <- [{"scram-sha-256", 6432}, {"md5", 6433},{"password", 6434}] do
+  for {key, port} <- [{"scram-sha-256", 6432}, {"password", 6434}] do
```

Comment:
Should probably have a function "wrapping" that list, and potentially reutrn all the connection options (hostname, port, database, etc) then we can merge the whole thing into the `ctx`, will probably make it smoother to add it in more places/will need less ad-hoc changes.